code/game/objects/structures/fluff/engineering/maintenance.dm

```diff
@@ -72,15 +72,22 @@ ABSTRACT_TYPE(/obj/structure/engineer_maintenance)
 	icon_state = "[panel_location]_[panel_type]_[icon_number]"
 
 /obj/structure/engineer_maintenance/attackby(obj/item/attacking_item, mob/user, params)
-	if(istype(attacking_item, /obj/item/powerdrill)) // these will always open the panel
+	if(istype(attacking_item, /obj/item/powerdrill) || istype(attacking_item, /obj/item/wrench)) // Any wrench is good, but we explicltly include the impact drill to send a helpful msg if they just have the screwdriver bit attached.
```

Comment:
There's a chance this might not include all wrenches? I don't know what borg wrenches or combiyltools are but if they're not a wrench subtype like the power drill, they might not get covered. Might be better to use iswrench() in this check?